changelog/pending/20240919--engine--add-input-keys-from-diff-during-import-step-to-warning-and-error-message.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: engine
+  description: "Add input keys from diff during import step to warning and error message."
```

```suggestion
  description: "Add input keys from diff during import step to warning and error message"
```

There shouldn't be any trailing dots on the changelog message.  See https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md#changelog-messages.

---

updated